src/node_mgmt/commands.py

```diff
@@ -180,14 +180,17 @@ def _set_address_holykell(mod: minimalmodbus.Instrument, result: dict,
         try:
             mod.write_register(80, target_slave_id, 0, 6)
             sleep(1)
+            # command to save changes
+            mod.address = target_slave_id
+            mod.write_register(64, 49087, 0, 6)
```

Comment:
Shouldn't it be register 78 (not 64)? Also where does 49087 come from? I think the value should just be 1 (if we follow the example in the docs)

Reply:
Oh ok I think I kind of see - there's another save command that we have lower down, which has now been moved up here. And we use the new approach only for the "alternative" model.

Reply:
yeah I'm not sure why we have that number, I checked the old doc with Olu and we also couldn't figure it out (but that doc doesn't have any info about how to change slave address anw) and this save part was done right after the one where we only do write_register to register `80`